code/game/gamemodes/gameticker.dm

```diff
@@ -99,13 +99,8 @@ var/global/datum/controller/gameticker/ticker
 	else
 		src.hide_mode = 0
 
-	if((master_mode=="random") || (master_mode=="secret"))
```

Comment:
what is all this you're doing with gamemodes?

Reply:
> what is all this you're doing with gamemodes?

At the moment, you are running the voting mode at startup. And since probality needs to be reworked into a normal form (in the new version of the probality configs, they do not work as needed). So I decided to remove this so that we wouldn't experience such a problem.